src/browser/BrowserService.cpp

```diff
@@ -216,6 +216,35 @@ QJsonObject BrowserService::getDatabaseGroups()
     return result;
 }
 
+QJsonArray BrowserService::getDatabaseEntries()
+{
+    auto db = getDatabase();
+    if (!db) {
+        return {};
+    }
+
+    Group* rootGroup = db->rootGroup();
+    if (!rootGroup) {
+        return {};
+    }
+
+    QJsonArray entries;
+    for (const auto& group : rootGroup->groupsRecursive(true)) {
+        if (group->isRecycled()) {
+            continue;
+        }
+
+        for (const auto& entry : group->entries()) {
+            QJsonObject jentry;
+            jentry["title"] = entry->title();
```

Comment:
You need to resolve references for this and the url call.

Reply:
Added `entry->resolveMultiplePlaceholders()` calls in the follow up commit.